library/std/src/panicking.rs

```diff
@@ -48,7 +48,7 @@ extern "C" {
     /// It cannot be `Box<dyn BoxMeUp>` because the other end of this call does not depend
     /// on liballoc, and thus cannot use `Box`.
     #[unwind(allowed)]
-    fn __rust_start_panic(payload: usize) -> u32;
+    fn __rust_start_panic(payload: *mut &mut dyn BoxMeUp) -> u32;
```

Comment:
Is the comment right above this no longer true?

```rust
/// `payload` is actually a `*mut &mut dyn BoxMeUp` but that would cause FFI warnings.
```

Reply:
Looks like it.^^ The build passes, after all...

I'll remove the comment.